src/casp/bq_scripts/extract_minibatch_to_anndata.py

```diff
@@ -116,17 +128,66 @@ def assign_uns_metadata(anndata, json_string):
         anndata.uns[key] = val
 
 
-def extract_minibatch_to_anndata(project, dataset, extract_table_prefix, start_bin, end_bin, output):
+def add_cell_type_info(
+    dataset: str, extract_table_prefix: str, bucket_name: str, adata: "ad.AnnData", cell_types: list
+):
+    filename = "all_cell_types.csv"
+    source_blob_name = f"{dataset}_{extract_table_prefix}_info/{filename}"
+
+    if not os.path.exists(filename):
+        utils.download_file_from_bucket(
+            bucket_name=bucket_name, source_blob_name=source_blob_name, destination_file_name=filename
+        )
+
+    adata.obs["cell_type"] = cell_types
+    df = pd.read_csv(filename, index_col=False)
+    adata.obs.cell_type = pd.Categorical(adata.obs.cell_type.values, categories=df["cell_type"].values)
+
+
+def add_expressed_genes_layer_mask(
```

Comment:
Rename `expressed` to `measured`, here and elsewhere where applicable.

An expressed gene in a cell is gene that produces at least one mRNA copy, i.e. we have measured it and it had > 0 counts.
A measured gene does not need to be expressed, i.e. we could have measured it and some or all cells had 0 counts of that gene.